go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/vsphere-csi-driver/v2
 
-go 1.19
+go 1.17
```

Comment:
We should not downgrade go version right ?

Reply:
not intentional, fixed that in latest commit